test/e2e/e2e.go

```diff
@@ -99,36 +99,17 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 		}
 	}
 
-	// In large clusters we may get to this point but still have a bunch
-	// of nodes without Routes created. Since this would make a node
-	// unschedulable, we need to wait until all of them are schedulable.
-	framework.ExpectNoError(framework.WaitForAllNodesSchedulable(c, framework.TestContext.NodeSchedulableTimeout))
```

Comment:
doesn't the skip have to go inside WaitForAllNodesSchedulable? there are lots of individual tests that call that

Reply:
That is an interesting question. I was initially thinking to place it outside in order to limit the scope of this to _just_ startup behavior.  It could become blocking for someone if the tests themselves are doing that.

However, that makes the flag a bit more complicated. Do we really want to skip that logic in a test which is specifically calling the method? Even if the cluster started in a known good state (hence you may provide the flag to skip the startup waiting), if a test tears down a node and wants to wait for it to come back, you really do want to wait then, not skip it like you did at startup.